test/unit_test/api/apps/services/test_delete_datasets.py

```
@@ -1,215 +1,250 @@
+        ),
+    )
+    _stub(
+        monkeypatch,
+        "api.db.services.file_service",
+        FileService=SimpleNamespace(filter_delete=file_filter_delete),
+    )
+    _stub(
+        monkeypatch,
+        "api.db.services.knowledgebase_service",
+        KnowledgebaseService=SimpleNamespace(
+            get_or_none=lambda id, tenant_id: kb,
```

Ruff (0.16.1)

[error] Lambda argument `id` is shadowing a Python builtin

(A006)